### ISSUE TITLE
CloudRunServiceNode cannot be cast to CloudRunTreeRootNode 

### ISSUE THREAD

**Issue by fsladkey** [GoogleCloudPlatform/cloud-code-intellij#2833]
## CloudRunServiceNode cannot be cast to CloudRunTreeRootNode 

OS: Windows 10
IDE Type: IntelliJ IDEA
IDE Version: 2020.2.2
Plugin Version: 20.9.1-202

```
class com.google.cloud.tools.intellij.cloudrun.serviceview.node.CloudRunServiceNode cannot be cast to class com.google.cloud.tools.intellij.cloudrun.serviceview.node.CloudRunTreeRootNode (com.google.cloud.tools.intellij.cloudrun.serviceview.node.CloudRunServiceNode and com.google.cloud.tools.intellij.cloudrun.serviceview.node.CloudRunTreeRootNode are in unnamed module of loader com.intellij.ide.plugins.cl.PluginClassLoader @18683f31)
error.stacktrace
java.lang.ClassCastException: class com.google.cloud.tools.intellij.cloudrun.serviceview.node.CloudRunServiceNode cannot be cast to class com.google.cloud.tools.intellij.cloudrun.serviceview.node.CloudRunTreeRootNode (com.google.cloud.tools.intellij.cloudrun.serviceview.node.CloudRunServiceNode and com.google.cloud.tools.intellij.cloudrun.serviceview.node.CloudRunTreeRootNode are in unnamed module of loader com.intellij.ide.plugins.cl.PluginClassLoader @18683f31)
	at com.google.cloud.tools.intellij.cloudrun.serviceview.node.CloudRunServiceNode$refreshNode$1.invokeSuspend(CloudRunServiceNode.kt:42)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(Dispatched.kt:241)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:971)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:841)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:452)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:744)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$9(IdeEventQueue.java:451)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:802)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:505)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

**Comment by fsladkey** [comment]
Kamino cloned this issue to [GoogleCloudPlatform/cloud-code-intellij-internal](https://github.com/GoogleCloudPlatform/cloud-code-intellij-internal/issues/2925)

**Comment by peterlin741** [comment]
Fixed in `20.10.1`
